test/index.js

```diff
@@ -202,3 +217,34 @@ describe('proxy server rows', function () {
     await checkResponseCode(http, 'http://localhost:8008/solr/select?q=fhqwhagads', 200)
   })
 })
+
+describe('proxy server start', function () {
+  var proxy
+  var solrTestDouble
+
+  beforeEach(function () {
+    proxy = SolrProxy.start(null, { maxStart: 1000 })
```

Comment:
Can we use a non-default value here to make sure it actually takes effect? Maybe change this to 2000 and then the test case below that is 1001 can be 2001 instead, and the test case that is 100 can be 1500? Something like that?